### PR TITLE
feat(ui): Add language support for kopiaUI

### PR DESCRIPTION
**Pull request by lupusA**
Hi, 

this PR adds the ability to support localisation in kopia-UI. The PR contains the following changes:

1. Updates the UI-Preferences interface to store the selected language

This PR has a dependency to https://github.com/kopia/htmlui/pull/241

Feedback is welcomed.

### CODE REVIEW

File: site/content/docs/Localisation/_index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
```
Comment (jkowalski) [inline site/content/docs/Localisation/_index.md:1]: i would move doc changes to separate PR to be merged closer to the release.

File: internal/serverapi/serverapi.go [hunk 2]
```diff
@@ -294,4 +294,5 @@ type UIPreferences struct {
 	Theme                  string `json:"theme"`                  // Specifies the theme used by the UI
 	FontSize               string `json:"fontSize"`               // Specifies the font size used by the UI
 	PageSize               int    `json:"pageSize"`               // A page size; the actual possible values will only be provided by the frontend
+	Language               string `json:"language"`               // Specifies the language used by the UI
```
Comment (jkowalski) [inline internal/serverapi/serverapi.go:297]: do we want to apply some default when returning the value or is empty string going to be enough?

**Review by jkowalski — APPROVED** [review]

File: internal/serverapi/serverapi.go (hunk 2 above)
Comment (lupusA) [reply internal/serverapi/serverapi.go:297]: It is handled by KopiaUI currently. If the value is empty it defaults to "en". However, we can think about setting a default here.

File: site/content/docs/Localisation/_index.md (hunk 1 above)
Comment (lupusA) [reply site/content/docs/Localisation/_index.md:1]: I will split up the pr into two separate ones.

**Review by jkowalski — APPROVED** [review]
approving this one to unblock testing, but please move docs out before merging

**Comment by lupusA** [comment]
Hi @jkowalski, 

i moved the doc to a different PR. This pr then only contains the change for the UI-Preference interface. 

Cheers,

**Review by jkowalski — APPROVED** [review]

**Pull request merged**